javascript_nodejs/14.nlp-with-dispatch/dialogs/weather/index.js

```diff
@@ -37,10 +37,10 @@ class Weather {
      * @param {Object} context context object
      */
     async onTurn(context) {
-        // make call to LUIS recognizer to get home automation intent + entities
+        // Call weather LUIS model.
         const weatherResults = await this.luisRecognizer.recognize(context);
         const topWeatherIntent = LuisRecognizer.topIntent(weatherResults);
-        // get location entity if available.
+        // Get location entity if available.
         const locationEntity = (LOCATION_ENTITY in weatherResults.entities)?weatherResults.entities[LOCATION_ENTITY][0] : undefined;
         const locationPatternAnyEntity = (LOCATION_PATTERNANY_ENTITY in weatherResults.entities)?weatherResults.entities[LOCATION_PATTERNANY_ENTITY][0]:undefined;
         // depending on intent, call turn on or turn off or return unknown
```

Comment:
grammar here reads oddly. Consider adding "Turn On" or "Turn Off" to make it read a bit more clearly.

Reply:
Fixed.